apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/MySqlStressTests.cs

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#if NET45 || NET451 || NET452
+#if NET452
```

Comment:
What about NETCOREAPP2_0 here?

Reply:
This is mysql which isnt working for netcore 2.0